core/modules/layout/layout.admin.inc

```diff
@@ -94,7 +94,34 @@ function layout_list_page() {
     array('data' => t('Operations'), 'class' => array('layout-operations')),
   );
 
+  $help = '<p>' . t('As a page is rendered, a <em>Layout</em> is selected as follows.') . '</p>';
+  $help_items = array();
+  $help_items[] = array(
+    'data' => 'The <em>Path</em> for the page is compared to those in the alphabetical list here, and <em>Visibility conditions</em> are evaluated.',
```

Comment:
It seems like all of the text except for this is in a t(function). Is there a reason that this line is not?

Reply:
Nope, good catch. It should be. Will make that change.